TFT/src/User/API/parseACK.c

```diff
@@ -1179,6 +1179,10 @@ void parseACK(void)
       {
         infoMachineSettings.onboardSD = ack_value();
       }
+      else if (ack_seen("Cap:MULTI_VOLUME:"))
```

Comment:
as for Cap:SDCARD, a sort of check on infoSettings.onboard_sd could be done. Maybe onboard_sd renamed to onboard_media

Reply:
Why? What would it prevent? Who would benefit from this check?

> as for Cap:SDCARD, a sort of check on infoSettings.onboard_sd could be done. 

Why? What would it prevent? Who would benefit from this check?

> Maybe onboard_sd renamed to onboard_media

It's debatable... You have to enable onboard SD to be able to reach onboard USB. There's no way to have USB enabled without the SD enabled.
Anyway, Marlin refers to onboard USD also as "SD Card". Check the message Marlin gives when USB is mounted.
So let's keep the naming synced.